raw_data/preprocessed/africa/zimbabwe_english.txt

```diff
@@ -0,0 +1,486 @@
+PART I - PRELIMINARY
+
+Short title
+(1) This Act may be cited as the Environmental Management Act [Chapter 20:27].
```

Comment:
@danpdraper so upon rechecking the link to the legislation (https://www.fao.org/faolex/results/details/en/c/LEX-FAOC047834), I have discovered that they updated the legislation from when I first created this script. I know this because I have kept the Zimbabwe legislation open in a google chrome tab from when I was working on it, and the url is the same, however the legislation now looks different. I have attached the older copy of the legislation that this script was based on here. Interestingly, this copy does not include sub-section 2 for Article 1.

However, I am wondering if you want me to rewrite the script based on the new legislation now? The content looks mostly the same, but I'm sure there will be some formatting differences. Updating may become problematic if they the legislation keeps getting updated however. 

[zim47834.pdf](https://github.com/danpdraper/eia/files/7991913/zim47834.pdf)

Reply:
I downloaded the file that you attached. I see sub-section 2 of article 1 in that file. I have attached a screenshot with sub-section 2 crudely highlighted.

As I've been reviewing the preprocessed legislation, I have been cross-referencing the sub-article numbers in the preprocessed legislation against the original legislation to ensure that they are all there. This is the only article in which I encountered a missing sub-article.

Furthermore, the FAOLEX page says that the copy of the legislation is a consolidation as of July 1st, 2005, so the copy should not be changing through time.

![Screenshot 2022-02-04 3 52 52 PM](https://user-images.githubusercontent.com/8284114/152618749-9289d00f-da57-4a18-a235-b507673ed77d.png)

Reply:
Sorry, I attached the wrong file. This is the version of the Zimbabwe legislation the preprocessing script was created based on: [2027updated.pdf](https://github.com/danpdraper/eia/files/8006967/2027updated.pdf)

The Internet Archive Wayback Machine does show my version of the legislation under the same link in July 2021: [https://web.archive.org/web/20210101000000*/http://extwprlegs1.fao.org/docs/pdf/zim47834.pdf](url) and I am quite sure it was changed between when I wrote the code and now.

That being said I do think sub-section 2 should be added, just wondering if you would like me to re-do the rest of the legislation based on the newly formatted Zimbabwe legislation.

Reply:
Yes, that's definitely a different file. Please amend the preprocessing script to accommodate the more recent version, if for no other reason than subsection [2] of section 1 is referenced elsewhere in the legislation (both the older version and the more recent version).

I just finished reviewing the rest of the legislation and the preprocessing script. I do not imagine that you will need to change very much to accommodate the more recent version of the legislation, but please ensure that your amendments respond to my comments.